### ISSUE TITLE
ICOc Command Line Tool Does Not Work with Python 3.12

### ISSUE THREAD

**Issue by sanssecours** [MyTooliT/ICOc#46]
# Description

ICOc does not start, if we use Python 3.12 to execute the tool.

# Steps to Reproduce

1. Install ICOc
2. Start ICOc:

   ```sh
   icoc
   ```

# Expected Result

ICOc lists the currently available sensor devices

# Actual Result

- IOC fails without any output to the screen
- Content of `cli.log`:

   ```
   2023-11-09 13:46:52,403 Initialized logger
   2023-11-09 13:46:52,924 Initialized CAN class
   2023-11-09 13:46:52,924 Measurement Channel 1: 1
   2023-11-09 13:46:52,924 Measurement Channel 2: 0
   2023-11-09 13:46:52,924 Measurement Channel 3: 0
   2023-11-09 13:46:52,925 STH Name: 
   2023-11-09 13:46:52,925 Bluetooth Address: 0
   2023-11-09 13:46:52,925 Connect to STH: False
   2023-11-09 13:46:52,926 Run Time: 0 s
   2023-11-09 13:46:52,926 Prescaler: 2
   2023-11-09 13:46:52,926 Acquisition Time: 8
   2023-11-09 13:46:52,926 Oversampling Rate: 64
   2023-11-09 13:46:52,926 Reference Voltage: VDD
   2023-11-09 13:46:52,926 Data Sets: 3
   2023-11-09 13:46:52,926 Sensors: M1: S1
   ```

# System Information

OS: Windows 10
Python Version: `3.12`

**Comment by sanssecours** [comment]
As far as I can tell the reason for this problem seems to be the [`windows-curses`](https://github.com/zephyrproject-rtos/windows-curses/issues/48) library.

**Comment by sanssecours** [comment]
This issue should be fixed in `windows-curses` version [`v2.3.3`](https://github.com/zephyrproject-rtos/windows-curses/releases/tag/v2.3.3).
